etc/init.d/orchestrator-agent.bash

```diff
@@ -60,21 +65,39 @@ case "$1" in
     PID=$(cat $PIDFILE)
     cd $DAEMON_PATH
     if [ -f $PIDFILE ]; then
-      kill -HUP $PID
-      printf "%s\n" "Ok"
+      kill -TERM $PID
       rm -f $PIDFILE
+      # Wait for orchestrator to stop otherwise restart may fail.
```

Comment:
Change to "Wait for orchestrator-agent to stop..."

Reply:
ok fixed